onnxruntime/core/providers/cuda/gpu_data_transfer.h

```diff
@@ -18,6 +18,7 @@ enum CUDAStreamType : int {
 class GPUDataTransfer : public IDataTransfer {
  public:
   GPUDataTransfer();
+  ~GPUDataTransfer();
```

Comment:
Destructor should never throw

Reply:
make sense! was copying the two lines from CudaExecutionProvider. changed it to not throw now.